### ISSUE TITLE
not compatible with Chinese and Japanese

### ISSUE THREAD

**Issue by chenxuuu** [spectresystems/spectre.console#150]
output extea lines

![图片](https://user-images.githubusercontent.com/10357394/100303365-4bbbb900-2fd7-11eb-896d-1bd3e736bce1.png)

![图片](https://user-images.githubusercontent.com/10357394/100303322-39417f80-2fd7-11eb-8081-6b75c3144541.png)

**Comment by sdutwsl** [comment]
In genaral, call it [CJK Unified Ideographs](https://en.wikipedia.org/wiki/CJK_Unified_Ideographs).

**Comment by patriksvensson** [comment]
@chenxuuu I have no idea why this happens. I first suspected that we get the wrong width for the CJK characters (i.e. 1 instead of 2), but there is no shift in horizontal spacing so this seems to be something else.

What terminal and OS are you running? What is the font you're using?

Also, could you post the changes you've made to a GitHub Gist or similar?

**Comment by chenxuuu** [comment]
I tested it with `powershell`(windows), `mintty`(windows) and `MobaXterm`(ssh to my linux machine)

output extra lines

code: https://gist.github.com/chenxuuu/3186c111558355825b6fef210cc92cff

```log
          Square                      Square           
  ┌──────────┬──────────┐     ┌──────────┬──────────┐  
  │ test     │ Header 2 │     │ 测试     │ Header 2 │  
  ├──────────┼──────────┤                              
  │ test     │     Cell │     ├──────────┼──────────┤  
  │ 11       │       22 │     │ 测试     │     Cell │  
  ├──────────┼──────────┤                              
  │ Footer 1 │ Footer 2 │     │ 11       │       22 │  
  └──────────┴──────────┘     ├──────────┼──────────┤  
                              │ Footer 1 │ Footer 2 │  
                              └──────────┴──────────┘  
                                                       

```

**Comment by patriksvensson** [comment]
@chenxuuu Looks like there's a bug in the `Panel` rendering. Will take a look tonight and see if I can solve it for you.
I think I know what it is.

![image](https://user-images.githubusercontent.com/357872/100430899-acaac600-3097-11eb-8889-a47d0a674f6a.png)

**Comment by patriksvensson** [comment]
@chenxuuu I just merged a fix for this which will be available in v0.35 or the next preview version if you don't want to wait.
